test/DFPerfScenariosV1/Tests/Counter.cs

```diff
@@ -14,7 +14,7 @@ namespace DFPerfScenarios
     public static class CounterTest
 	{
 		[FunctionName("StartCounter")]
-		public static async Task<HttpResponseMessage> Start(
+		public static HttpResponseMessage Start(
```

Comment:
Made this extra change to fix a build warning.

Reply:
Huh, interesting. It's not immediately clear to me why this change was needed, but I won't block on it

Reply:
Not required - just cleaning up noise as I find issues getting the main objectives working.